src/transformers/models/paligemma/modeling_paligemma.py

```diff
@@ -299,11 +300,18 @@ def _init_weights(self, module):
 class PaliGemmaForConditionalGeneration(PaliGemmaPreTrainedModel, GenerationMixin):
     def __init__(self, config: PaliGemmaConfig):
         super().__init__(config)
-        self.vision_tower = AutoModel.from_config(config=config.vision_config)
+
+        vision_config = deepcopy(config.vision_config)
+        text_config = deepcopy(config.text_config)
+        if hasattr(config, "quantization_config"):
+            vision_config.quantization_config = config.quantization_config
+            text_config.quantization_config = config.quantization_config
```

Comment:
We don't want to have this in the model config. Otherwise, we would have to do it for every model. Also, we shouldn't need to do that as we quantize the model from the top level. Maybe we can propagate the `quantization_config` in `from_pretrained` ?

Reply:
The problem is that model that are composed from multiple sub models that calls from_config are not aware that thy should not use deepspeed.zero.init as it is the case of paligemma.

Reply:
`quantization_config` is already passed to the model class via `from_pretrained`, but the sub-models are instantiated using `from_config`, which does not include it. Perhaps we can propagate the quantization information using a context manager.